nodes/Expr.py

```diff
@@ -15,3 +15,16 @@
 print(1)
 (100 * 42)
 101 * 43
+
+0
+6.0
```

Comment:
Please add a comment here saying "extra examples" (because these examples aren't discussed in the docstring).